module/VuFind/src/VuFind/Http/CachingDownloader.php

```diff
@@ -187,15 +187,16 @@ public function download(
      * Download a resource using the cache in the background,
      * including decoding for JSON.
      *
-     * @param string $url    URL
-     * @param array  $params Request parameters (e.g. additional headers)
+     * @param string    $url         URL
+     * @param array     $params      Request parameters (e.g. additional headers)
+     * @param bool|null $associative Sent to json_decode
      *
      * @return stdClass
      */
-    public function downloadJson($url, $params = [])
+    public function downloadJson($url, $params = [], $associative = null)
```

Comment:
This seemed small enough to include in this PR, but if we want to make it more complex (i.e. pass an some structure instead that could contain $associative any any other params to pass to json_decode) then I can submit it separately.